### PR TITLE
Include seven `<Fieldsets />` on p003

### PR DESCRIPTION
**Pull request by Andresbl123**
In this PR, the following changes have been made:

1. A new `<FinancialReportingSections />` component has been created that contains the structure and content of the financial reporting sections. This component uses the `<Stack />` component to organise the elements in a column direction and the `<Grid />` component to organise the elements in a grid.

2. The `<Fieldset />` component has been used to encapsulate each financial reporting section. Each `<Fieldset />` contains a title and a description.

3. The Text component has been used to provide content for each `<Fieldset />`.

4. A new `<FinancialReporting />` component has been created that encapsulates the `<FinancialReportingSections />` component within the `<ContainerSections />` component.

image

.

### CODE REVIEW

File: src/pages/home/financialReporting/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Stack, inube, Text, Grid } from "@inube/design-system";
+import { Fieldset } from "@src/components/data/Fieldset";
```
Comment (novadeveloper14) [inline src/pages/home/financialReporting/index.tsx:2]: Adjust to the standard for imports in this team:
(1) External components
(space)
(2) Internal components which use aliases
(space)
(3) Internal component without aliases

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/pages/home/financialReporting/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/pages/home/financialReporting/index.tsx]: resuelto

**Review by novadeveloper14 — APPROVED** [review]

File: src/components/data/Fieldset/styles.ts [hunk 2]
```diff
@@ -6,7 +6,8 @@ export const StyledContainerFieldset = styled.div`
   border-width: 2px;
   border-style: solid;
   border-color: ${({ theme }) =>
-    theme?.color?.stroke?.gray?.regular || inube.color.stroke.gray.regular};
+    theme?.color?.stroke?.divider?.regular ||
```
Comment (carin91) [inline src/components/data/Fieldset/styles.ts:9]: why do you use the divider block for the border?

File: src/pages/home/financialReporting/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <Stack direction="column" gap={inube.spacing.s500}>
+      <Stack direction="column">
+        <Fieldset title="Estado" descriptionTitle="Gestión Comercial">
+          <Text></Text>
+        </Fieldset>
+      </Stack>
+      <Grid templateColumns="repeat(2,1fr)" gap="s200" autoRows="auto">
+        {configData.map((data) => (
+          <Fieldset title={data.title} descriptionTitle={data.descriptionTitle}>
```
Comment (carin91) [inline src/pages/home/financialReporting/index.tsx:17]: The fieldset is a prop of the children, is not created/defined on the page. So, to define the structure it is ok, however, every new component created would have its fieldset. Don't you think?

File: src/pages/home/financialReporting/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { Stack, inube, Text, Grid } from "@inube/design-system";
```
Comment (carin91) [inline src/pages/home/financialReporting/index.tsx:1]: The location of this file should be evaluated, however, remember that the home folder will be deleted (this is included in the cb27)

File: src/App.tsx [hunk 5]
```diff
@@ -37,7 +37,7 @@ const router = createBrowserRouter(
       <Route path="login/*" element={<LoginRoutes />} />
       <Route path="board/*" element={<BoardRoutes />} />
       <Route path="logout" element={<LogOut />} />
-      <Route path="sections" element={<ContainerSections />} />
+      <Route path="financial-reporting" element={<FinancialReporting />} />
```
Comment (carin91) [inline src/App.tsx:40]: the financial-reporting will be a path of the board, this is ok to verify the rendered page, however, the cb27 includes a different location for this page. We need to evaluate the best location for this page.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/home/financialReporting/index.tsx (hunk 3 above)
Comment (Andresbl123) [reply src/pages/home/financialReporting/index.tsx:17]: Claro si sr, por ahora esto esta para definir el layout  de la pagina a medida que se ingreses los compontes ya es responsabilidad del mismo mirar como será su contenedor, pero se dejo así para indicar el outlet donde se van a colocar lo que se quiere renderizar según la mock del diseño

File: src/components/data/Fieldset/styles.ts (hunk 2 above)
Comment (Andresbl123) [reply src/components/data/Fieldset/styles.ts:9]: resuelto

File: src/pages/home/financialReporting/index.tsx (hunk 4 above)
Comment (Andresbl123) [reply src/pages/home/financialReporting/index.tsx]: validado

File: src/App.tsx (hunk 5 above)
Comment (Andresbl123) [reply src/App.tsx:39]: validado

**Pull request merged**